src/MarginTrading.SettingsService/Settings/ServiceSettings/DbSettings.cs

```diff
@@ -4,9 +4,14 @@ namespace MarginTrading.SettingsService.Settings.ServiceSettings
 {
     public class DbSettings
     {
+        public string StorageMode { get; set; }
+        
         [AzureTableCheck]
         public string LogsConnString { get; set; }
         [AzureTableCheck]
         public string MarginTradingConnString { get; set; }
```

Comment:
In that case may be better to rename to AzureConnectionString? )

Reply:
+